BICI_Psoc_workspace/Fingertip_V3_acc.cydsn/main.c

```diff
@@ -88,6 +169,17 @@ int main(void)
                     i2cReadBuffer[i+59] = CapSense_dsRam.snsList.row11[i].raw[0]; 
                     i2cReadBuffer[i+63] = CapSense_dsRam.snsList.row12[i].raw[0]; 
                 }
+                for(unsigned int i=0; i<1; ++i) // Proxi

```

Comment:
- Don't use a while loop for one element...
- We did not include proxi because it doesn't work in this design

Reply:
but it might work, right? and it doesn't hurt at the moment

Reply:
It adds more load on the communication channel that is already overloaded, we don't need this. If you want to keep this please at least comment it.
And don't do a loop for 1 iteration

Reply:
I think we should keep it to test the network at least, we can remove it later if you want in case we totally give up on proxi. In any case, I removed the loop.